content/team/antonio-gamez-diaz/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Antonio Gámez Díaz
+description: ""
+roles: ["guest"]
+skills: ""
+location: ""
+
+# Social Media
+twitter: ""
+website: ""
+facebook: ""
+linkedin: ""
```

Comment:
Hi @tiffanyfay, could you please, when you have some time, no rush at all, change it with the following? Or perhaps you want me to send a PR ?

```
description: "Member of Technical Staff"
location: "Sevilla, Spain"
linkedin: "antoniogamezdiaz"
github: "antgamdia"
```

Reply:
@antgamdia Yep! Will merge once the checks pass!